build_tools/cmake/build_android.sh

```diff
@@ -21,18 +20,31 @@ fi
 
 ANDROID_ABI=$1
 
-ROOT_DIR=$(git rev-parse --show-toplevel)
-
 CMAKE_BIN=${CMAKE_BIN:-$(which cmake)}
 
-"${CMAKE_BIN?}" --version
+"${CMAKE_BIN}" --version
+"${CC}" --version
+"${CXX}" --version
 ninja --version
+echo "Android NDK path: ${ANDROID_NDK}"
+
+ROOT_DIR=$(git rev-parse --show-toplevel)
+cd ${ROOT_DIR}
 
-cd ${ROOT_DIR?}
+# BUILD the iree-import-tflite binary for importing models to benchmark from
+# TFLite flatbuffers.
+cd "${ROOT_DIR}/integrations/tensorflow"
+BAZEL_CMD=(bazel --noworkspace_rc --bazelrc=build_tools/bazel/iree-tf.bazelrc)
+BAZEL_BINDIR="$(${BAZEL_CMD[@]} info bazel-bin)"
+"${BAZEL_CMD[@]}" build //iree_tf_compiler:iree-import-tflite \
+      --config=generic_clang \
+      --config=remote_cache_tf_integrations
```

Comment:
I see errors on CI with this change merged like https://buildkite.com/iree/iree-android-arm64-v8a/builds/5255#8a2feee2-ed0f-4b11-9466-191d58f53c1a

```
[[14:38:32 UTC] BAZEL_CMD=(bazel --noworkspace_rc --bazelrc=build_tools/bazel/iree-tf.bazelrc)
[[[14:38:32 UTC] bazel --noworkspace_rc --bazelrc=build_tools/bazel/iree-tf.bazelrc info bazel-bin
FATAL: $USER is not set, and unable to look up name of current user: (error: 0): Success
[[14:38:32 UTC] BAZEL_BINDIR='ERROR: The project you'\''re trying to build requires Bazel 3.7.1 (specified in /usr/src/github/iree/integrations/tensorflow/.bazelversion), but it wasn'\''t found in /usr/bin.
 
You can install the required Bazel version via apt:
  sudo apt update && sudo apt install bazel-3.7.1
```

---

`build_tools/cmake/build_android.sh` should not take a dependency on Bazel at all IMO. I'd rather see a new script build the `iree-import-tflite` binary and use that from whatever CI is involved. Any Bazel dep will add build time, configuration complexity, and prevent use on Windows.

If the dependency stays, it should output more information in the associated scripts, such as the Bazel version (like how the CMake version is printed). This also isn't accurate now: https://github.com/google/iree/blob/f06c804f9ec79e16da7dcee1af81afd7b95a8bb1/build_tools/kokoro/gcp_ubuntu/cmake/android/build.sh#L35-L36

Reply:
agreed, no bazel deps there

Reply:
I think the simple fix is to change https://github.com/google/iree/blob/f06c804f9ec79e16da7dcee1af81afd7b95a8bb1/build_tools/buildkite/cmake/android/arm64-v8a/pipeline.yml#L10 

to also use the `frontends` image, but again... I have concerns about adding Bazel to this script.

Reply:
Sorry that I missed that build. See upthread for the various options discussed. I agree that the right way is to have another build create the artifact for usage elsewhere, but that requires the CI restructuring that was deprioritized. For that build in particular, I don't think we need to have Bazel at all because there's no reason it needs to also build the benchmarks. I'll switch it to use a different script

Reply:
Oh I know why I didn't spot this. The Buildkite pipeline is running the script build_tools/kokoro/gcp_ubuntu/cmake/android/build.sh, which I foolishly assumed was only for the Kokoro build :-P

Reply:
Fixed in #7541